daiquiri/query/templates/query/query_plot_histogram.html

```diff
@@ -1,48 +1,49 @@
 {% load i18n %}
 
-<div class="row">
-    <div class="col-md-4">
-    	<label class="control-label" for="plot_x">{% trans 'Column'%}</label>
-        <div class="form-inline" id="plot_x" ng-class="{'has-error': service.plot.errors.x }">
-            <select class="form-control-sm select-value-md" ng-model="service.plot.values.x" ng-change="service.plot.errors.x = null">
-                <option value="{$ option.name $}"
-                    ng-repeat="option in service.plot.columns">
-                    {$ option.name $}
-                </option>
-            </select>
-            <ul class="help-block list-unstyled">
-                <li ng-show="service.plot.errors.x">
-                    <span ng-show="service.plot.errors.x == 'empty'">
-                        {% trans 'No column selected' %}
-                    </span>
-                    <span ng-show="service.plot.errors.x != 'empty'">
-                        {% blocktrans with datatype='{$ service.plot.errors.x $}' %}
```

Comment:
Your removed the error catching, is that because safe columns needs to be declared in the settings?

Reply:
Yes. From now on, only column types that are 'plottable' can be selected via the web interface. The supported data types are declared by `validtypes` in `plot.js`